docsets/solarus/docset.json

```diff
@@ -0,0 +1,20 @@
+{
+    "name": "Solarus",
+    "version": "1.6.0",
+    "archive": "solarus.docset",
+    "author": {
+        "name": "Santiago Itzcoatl",
+        "link": "https://twitter.com/santiagoitzcoat"
+    },
+    "aliases": ["Solarus, Solarus-Engine, Solarus-Game-Engine, ARPG"],
+
+    "specific_versions": [
+        {
+            "_comment": "Version log:"
+        },
+        { 
+            "version": "1.6.0",
+            "archive": "versions/1.6.0/solarus.docset.tgz",
```

Comment:
You need to add the docset at this path (`versions/1.6.0/solarus.docset.tgz`), or remove this entry.